src/cs/LionWeb.Integration.WebSocket.Tests/WebSocketServerTestBase.cs

```diff
@@ -59,4 +59,16 @@ public void StopServer()
     {
         _webSocketServer.Stop();
     }
+
+    protected void WaitForSent(int delta = 1)
```

Comment:
Maybe `numberOfMessages` instead of `delta`?

Reply:
I like this more than `targetCount` from above; please unify over all `WaitForXxx()` methods